kubernetes/sourcegraph.yaml.erb

```diff
@@ -92,28 +59,37 @@ spec:
             timeoutSeconds: 3
             periodSeconds: 5
       volumes:
-        - name: sourcegraph-secrets
-          hostPath:
-            path: /opt/share/kubernetes/secrets/sourcegraph
-            type: Directory
+        - name: sourcegraph-config
```

Comment:
I don't see this volume actually mounted into the container anywhere

Reply:
Good catch, I meant to replace `sourcegraph-secrets` further up with this! I've done so in https://github.com/ocf/sourcegraph/pull/4/commits/9f13ef48ffcf223df7f28f0151bebb36d792aa38.